docs/product/roadmap.md

```diff
@@ -0,0 +1,65 @@
+Product roadmap
+===================
+
+## Overview
+This is a high-level overview. For more detail, see our [story map](https://www.google.com/url?q=https://app.mural.co/t/gsa6/m/gsa6/1647625636085/7825fa5ead63ebb1b65eefab5ea20d24a8ab7c59?wid%3D0-1647626493256%26sender%3Dua4d37dfba3f1e69e09078790&sa=D&source=docs&ust=1654713307298233&usg=AOvVaw2VRKptBGcaZgHol3V6vXpw) or [user flows chart](https://app.mural.co/t/gsa6/m/gsa6/1654109929324/5da173ea5f7ba75104e4e624451e58496a7164c8?sender=ua4d37dfba3f1e69e09078790). As with all agile product roadmaps, this is subject to change. The team revisits and adjusts the roadmap regularly. 
+
+## Product Vision
```

Comment:
It's very possible that folks reviewing this will be expecting something a bit more aspirational. There has been a lot of work on articulating the future vision of the FAC prior to us starting the project that may be worth consulting an/or incorporating here. Otherwise, these read a bit more like overall the goals of the full transition.

Reply:
Would love to pull in more aspirational language here. Honestly, I'm not aware of motivating factors beyond this!